### PR TITLE
add mcq scaffolding to mx3, mx4

### PR DESCRIPTION
**Pull request by StevenClontz**

### CODE REVIEW

File: source/04-MX/03.ptx [hunk 1]
```diff
@@ -28,86 +28,159 @@ Consider the following linear system with a unique solution:
         </introduction>
         <task>
             <p>
-Define
```
Comment (StevenClontz) [inline source/04-MX/03.ptx:31]: This activity (which is new to the 2023 edition and hadn't been taught before I think) was a bit of a dud and I ended up lecturing through it more or less. :-(

File: source/04-MX/04.ptx [hunk 2]
```diff
@@ -21,35 +21,155 @@ in terms of matrix multiplication.
...
+Which of these tweaks of the identity matrix
+yields a matrix that doubles the third row of <m>A</m> when
+left-multiplying? (<m>2R_3\to R_3</m>)
 <me>
  \left[\begin{array}{ccc} \unknown &amp; \unknown &amp; \unknown \\ \unknown &amp; \unknown &amp; \unknown \\ \unknown &amp; \unknown &amp; \unknown \end{array}\right]
  \left[\begin{array}{ccc} 2 &amp; 7 &amp; -1 \\ 0 &amp; 3 &amp; 2 \\ 1 &amp; 1 &amp; -1 \end{array}\right]
 =
  \left[\begin{array}{ccc} 2 &amp; 7 &amp; -1 \\ 0 &amp; 3 &amp; 2 \\ 2 &amp; 2 &amp; -2 \end{array}\right]
 </me>
+    <ol label="A." cols="2">
```
Comment (StevenClontz) [inline source/04-MX/04.ptx:33]: Adding some choices here so folks can at least plug and chug out an answer, but still use inquiry to more cleverly guess which of the choices is most appropriate.

File: source/04-MX/04.ptx [hunk 3]
```diff
@@ -106,6 +226,20 @@ for some sequence of matrices <m>R_1,R_2,\dots,R_k</m>.
     </statement>
 </fact>
 
+<activity>
```
Comment (StevenClontz) [inline source/04-MX/04.ptx:229]: We ended up observing this anyway, but it's good to have on the books.

File: source/04-MX/04.ptx [hunk 4]
```diff
@@ -168,8 +302,30 @@ Check your work using technology.
     </statement>
 </activity>
 
-<sage language="octave">
-</sage>
+<sage language="octave"/>
+
+<activity>
```
Comment (StevenClontz) [inline source/04-MX/04.ptx:307]: Added activity closer to quizzes.

**Review by siwelwerd — APPROVED** [review]

**Pull request merged**